src/Microsoft.TestPlatform.Extensions.TrxLogger/Utility/Converter.cs

```diff
@@ -474,6 +474,8 @@ private CollectorDataEntry ToCollectorEntry(ObjectModel.AttachmentSet attachment
                 }
 
                 string sourceFile = uriDataAttachment.Uri.LocalPath;
+                
+                var rooted = (Path.GetFullPath(sourceFile) == sourceFile);
                 Debug.Assert(Path.IsPathRooted(sourceFile), "Source file is not rooted");
```

Comment:
heads up: you are not using `rooted` variable are.

Reply:
removed it.